### PR TITLE
new pandas cheat sheet fomats

### LINKED ISSUE #54910: DOC: pandas cheat sheet
### Pandas version checks

- [X] I have checked that the issue still exists on the latest versions of the docs on `main` [here](https://pandas.pydata.org/docs/dev/)


### Location of the documentation

https://github.com/pandas-dev/pandas/blob/main/doc/cheatsheet/Pandas_Cheat_Sheet.pdf

### Documentation problem

The pandas cheat sheets could be better if it was complemented with tools like google colab and streamlit

### Suggested fix for documentation

I have developed a repository with several cheat sheets in pdf, streamlit and google colab format, in case you are interested in improving the learning experience of this library.

| Topic         | PDF                                                                                                                                                                                                                                  | Streamlit                                                                                                                                                        | Google Colab                                                                                                                                                                                                                                   |
|---------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| pandas        | <a href="https://github.com/fralfaro/DS-Cheat-Sheets/blob/main/docs/files/pandas_cs.pdf" target="_parent"><img src="https://img.shields.io/badge/Open in PDF-%23FF0000.svg?style=flat-square&logo=adobe&logoColor=white"/></a>       | <a href="https://ds-cheat-sheets-pandas.streamlit.app/" target="_parent"><img src="https://static.streamlit.io/badges/streamlit_badge_black_white.svg"/></a>     | <a href="https://colab.research.google.com/github/fralfaro/DS-Cheat-Sheets/blob/main/docs/examples/pandas/pandas.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>         |}

> **Note**: The PDF format cheat sheets included here are authored by other contributors and have been used as sources of inspiration for the content presented.

### PR DESCRIPTION
**Pull request by fralfaro**
- [x] closes #54910 
- [x] Added an entry in the latest `doc/cheatsheet/` file if fixing a bug or adding a new feature.

### CODE REVIEW

File: doc/cheatsheet/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Pandas Cheat Sheet
+
+The Pandas Cheat Sheet was created using Microsoft Powerpoint 2013.
+To create the PDF version, within Powerpoint, simply do a "Save As"
+and pick "PDF" as the format.
+
+This cheat sheet was inspired by the [RStudio Data Wrangling Cheatsheet](https://www.rstudio.com/wp-content/uploads/2015/02/data-wrangling-cheatsheet.pdf),
+written by Irv Lustig, [Princeton Consultants](https://www.princetonoptimization.com/).
```
Comment (Dr-Irv) [inline doc/cheatsheet/README.md:8]:
```suggestion
This cheat sheet, originally written by Irv Lustig, [Princeton Consultants](https://www.princetonoptimization.com/), was inspired by the [RStudio Data Wrangling Cheatsheet](https://www.rstudio.com/wp-content/uploads/2015/02/data-wrangling-cheatsheet.pdf),

```
Without the footnotes, it reads as if I wrote the RStudio cheat sheet, so I reworded the sentence accordingly.

**Review by Dr-Irv — CHANGES_REQUESTED** [review]
Thanks for the PR.  I've suggested some alternative wordings.

File: doc/cheatsheet/README.md (hunk 1 above)
Comment (fralfaro) [reply doc/cheatsheet/README.md]: Thanks for the suggestion, I'll make the changes immediately.

**Review by Dr-Irv — APPROVED** [review]
thanks @fralfaro

**Pull request merged**